packages/modules/product/src/repositories/product.ts

```diff
@@ -13,28 +17,78 @@ export class ProductRepository extends DALUtils.mikroOrmBaseRepositoryFactory(
     super(...arguments)
   }
 
+  /**
+   * Identify the relations to load for the given update.
+   * @param update
+   * @returns
+   */
+  getRelationsToLoad(update: any[]): string[] {
```

Comment:
`update` is singular but it's an array it's misleading.

Reply:
Also seems to be `updates` in other methods see `deepUpdate`

Reply:
ahah it was still in progress, I ve changed the naming completely as I prefer explicit naming personally